ci-operator/jobs/openshift/console/openshift-console-master-presubmits.yaml

```diff
@@ -1,5 +1,195 @@
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-console-master.yaml
+              name: ci-operator-configs
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          limits:
+            cpu: 500m
+          requests:
+            cpu: 10m
+      serviceAccountName: ci-operator
+    trigger: ((?m)^/test( all| GUI-Tests-New-App),?(\s+|$))
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/GUI-Tests-OLM
+    decorate: true
+    name: pull-ci-openshift-console-master-GUI-Tests-OLM
+    rerun_command: /test GUI-Tests-OLM
+    skip_cloning: true
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --target=GUI-Tests-OLM
+        command:
+        - ci-operator
+        env:
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-console-master.yaml
+              name: ci-operator-configs
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          limits:
+            cpu: 500m
+          requests:
+            cpu: 10m
+      serviceAccountName: ci-operator
+    trigger: ((?m)^/test( all| GUI-Tests-OLM),?(\s+|$))
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/GUI-Tests-Service-Catalog
+    decorate: true
+    name: pull-ci-openshift-console-master-GUI-Tests-Service-Catalog
+    rerun_command: /test GUI-Tests-Service-Catalog
+    skip_cloning: true
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --target=GUI-Tests-Service-Catalog
+        command:
+        - ci-operator
+        env:
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-console-master.yaml
+              name: ci-operator-configs
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          limits:
+            cpu: 500m
+          requests:
+            cpu: 10m
+      serviceAccountName: ci-operator
+    trigger: ((?m)^/test( all| GUI-Tests-Service-Catalog),?(\s+|$))
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/Performance
+    decorate: true
+    name: pull-ci-openshift-console-master-Performance
+    rerun_command: /test Performance
+    skip_cloning: true
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/Performance-cluster-profile
+        - --target=Performance
+        - --template=/usr/local/Performance
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_TYPE
+          value: aws
```

Comment:
From what I can tell, `cluster-launch-src.yaml` can only be run with gcp, not aws.  I've found the playbook in the template `launch.yml` only exists in the ansible image here:  `playbooks/gcp/openshift-cluster/launch.yaml` but not here: `playbooks/aws/openshift-cluster/launch.yaml`